src/components/results/dynamicsimulation/timeseries/dynamic-simulation-result-series-chart.tsx

```diff
@@ -99,37 +99,45 @@ function DynamicSimulationResultSeriesChart({
                 }
                 action={
                     <>
-                        <ToggleButton
-                            sx={styles.plotScaleButton}
-                            size={'small'}
-                            value={'plotScale'}
-                            selected={plotScale}
-                            onChange={() => handlePlotScale(id)}
-                        >
-                            {plotScale ? (
-                                <Tooltip
-                                    title={intl.formatMessage({
-                                        id: 'DynamicSimulationPlotScaleDisable',
-                                    })}
-                                >
-                                    <FullscreenExitSharpIcon />
-                                </Tooltip>
-                            ) : (
-                                <Tooltip
-                                    title={intl.formatMessage({
-                                        id: 'DynamicSimulationPlotScaleEnable',
-                                    })}
-                                >
-                                    <FitScreenSharpIcon />
-                                </Tooltip>
-                            )}
-                        </ToggleButton>
+                        {plotScale ? (
```

Comment:
I think "plotScale" is misleading. Isn't it a focus on the plot which then takes the whole viewport ? I think this is clearer : 
```suggestion
                        {plotForegrounded ? (
```
(or simply plotFullViewport)